ils_middleware/tasks/sinopia/local_metadata.py

```diff
@@ -71,7 +71,9 @@ def create_admin_metadata(**kwargs) -> str:
 def new_local_admin_metadata(*args, **kwargs) -> str:
     "Add Identifier to Sinopia localAdminMetadata."
     jwt = kwargs.get("jwt")
-    group = kwargs.get("group")
+    resource = kwargs.get("resource", {})
+    group = resource.get("group")
+    editGroups = resource.get("editGroups")
```

Comment:
Small suggestion, could you change this `get` dict call to return an empty list if `editGroups` is null, i.e. `resource.get("editGroups", [])` that should make the `sinopia_doc` more resilient when saved back to Sinopia.

Reply:
Done.